src/main/resources/site/pages/default/default.js

```diff
@@ -1,77 +1,112 @@
-var stk = require('/lib/stk/stk');
+var thymeleaf = require('/lib/thymeleaf');
```

Comment:
Yes kill STA lib! 🤩 
Mene hvorfor `var` vi er es 7 (ish) nå const på libs og view?

Reply:
True. I can go through the code and fix that.